ibis/pandas/execution/selection.py

```diff
@@ -107,16 +119,21 @@ def compute_projection_column_expr(
         )
 
     data_columns = frozenset(data.columns)
-    additional_scope = {
-        t: map_new_column_names_to_data(
-            remap_overlapping_column_names(parent_table_op, t, data_columns),
-            data,
+    for t in op.root_tables():
```

Comment:
Are the ordering here guaranteed to be same as the orignal one?

Reply:
Yes these scopes are merged in order.